finite-supply.adoc

```diff
@@ -146,53 +146,53 @@ mempool—a small chance at a large amount can be worth more than a
 large chance at a small amount.
 ____
 
-To further put a wet blanket over our hopes for the future, if miners
-conduct fee sniping, it further incentivizes others to do the same,
+Throwing a wet blanket over our hopes for the future is the fact that if miners start
+conducting fee sniping, this will incentivize others to do the same,
 leaving even fewer honest miners. This could severely impair the
 overall security of Bitcoin. Harding goes on to list a few
-countermeasures that have been taken, for example using transaction
-lock time to restrict where in the blockchain the transaction may
+countermeasures that can be taken, such as relying on transaction
+time locks to restrict where in the blockchain the transaction may
 appear.
 
 So, given that the consensus on finite supply remains, the block
-subsidy will, thanks to
+subsidy will - thanks to
 https://github.com/bitcoin/bips/blob/master/bip-0042.mediawiki[BIP42]
-that fixed a very-long-term inflation bug ;), be zero from around year
-2140. Will the transaction fees after that be enough to secure the
-network? It's impossible to say, but we do know some things:
+which fixed a very-long-term inflation bug - get to zero around year
+2140. Will the transaction fees thereafter be enough to secure the
+network? It's impossible to say, but we do know a few things:
 
 * A century is a _long_ time from the Bitcoin perspective. If it is
-  still around, it has probably evolved enormously.
-* If an overwhelming economic majority finds it attractive to change
+  still around, it will have probably evolved enormously.
+* If an overwhelming economic majority finds it convenient to change
   the rules and introduce for example a perpetual annual 0.1% or 1%
   monetary inflation, the supply of bitcoin will no longer be finite.
-* With zero block subsidy and an empty or almost empty mempool, things
+* With zero block subsidy and an empty or nearly empty mempool, things
   can become shaky due to fee sniping.
 
-Since the transition to a fee-only economy is so far in the future, it
-might be wise to not jump to conclusions and try to fix the potential
-issues. For example, Peter Todd thinks there's a risk that Bitcoin's
-security budget won't be enough in the future and argued for a small
+Since the transition to a fee-only block reward is so far in the future, it
```

Comment:
This is a great edit! Fantastic!

Reply:
Thank you very much! I really appreciate you giving me constant feedback, it's the only way I have to know whether I'm doing a proper job or I'm going off track